tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,2 +1,12 @@
-frost-login testbed
-{{frost-login}}
+<div class="dummy-header">
+  {{#frost-link 'default' class='medium secondary'}}
+      Company default
```

Comment:
Probably no point in having the default, just the custom with Frost - I mean, literally nobody is going to use the default ;)